pkg/1password/cli.go

```diff
@@ -197,6 +197,30 @@ func (cli *Cli) RemoveUserFromGroup(ctx context.Context, group, user string) err
 	return nil
 }
 
+// AddUserToVault adds user to vault.
+func (cli *Cli) AddUserToVault(ctx context.Context, vault, user, permissions string) error {
+	args := []string{"vault", "user", "grant", "--vault", vault, "--user", user, "--permissions", permissions}
+
+	err := cli.executeCommand(ctx, args, nil)
+	if err != nil {
+		return fmt.Errorf("error adding user to vault: %w", err)
+	}
+
+	return nil
+}
+
+// RemoveUserFromVault removes user from vault.
+func (cli *Cli) RemoveUserFromVault(ctx context.Context, vault, user, permissions string) error {
```

Comment:
I'm not sure how to reflect this in the documentation or error messages, but remove can't remove a user if they're a member of a group that has permissions on the vault. eg: If John Smith is a member of the Administrators group, and the Administrators group has lots of permissions on a vault, `op` errors with "the accessor doesn't have any permissions".

I don't want to make extra work for you, so I'm just pointing out that this is an issue, not that you need to fix it. Maybe add a comment to inspire someone to add this functionality in the future?